hazelcast/src/main/resources/hazelcast-client-full-example.yaml

```diff
@@ -827,3 +827,30 @@ hazelcast-client:
       - file://User/test/sample.jar
     classNames:
       - test.sampleClassName
+
+  #
+  # ===== HAZELCAST METRICS CONFIGURATION =====
+  #
+  # Configuration element's name is "metrics".
```

Comment:
Same consideration as in `hazelcast-client-full-example.xml`.

Reply:
Done